alacritty.yml

```diff
@@ -338,6 +338,13 @@
   # See `cursor.style` for available options.
   #vi_mode_style: None
 
+  # Blinking cursors blink rate in milliseconds
+  #
+  # If a cursor is blinking, defines at what rate. This concerns the "Blinking*"
+  # cursor style above as well as escape sequences that makes the cursor start
+  # blinking.
+  #blink_rate: 500
```

Comment:
Or, instead of adding a separate option, set `blink_rate` to `0` to disable blinking.

One more thing: The unit of the value is ms, right? I'd suggest to rename the config option to `blink_period` or make this a proper rate where the default value is 2 Hz. Or, at least, mention somewhere in the comment that the value has the unit ms.

Reply:
Yes, setting `blink_rate` to 0 does disable blinking, I just forgot to say it in the comment, thanks for the reminder.

The fact that it's in ms is said on the first line of that comment. I personally find a value of ms more common than a frequency in Hz, I rarely see tools that have configuration options in Hz.

Reply:
Oh yeah, my bad. I was too focused on the paragraph, nevermind then.